.golangci.yaml

```
@@ -179,6 +179,10 @@ issues:
     - stylecheck
     text: "ST1016: methods on the same type should have the same receiver name"
     path: .*(api|types)\/.*\/conversion.*\.go$
+  - path: notes\.go
+    linters:
+    - forbidigo
+    text: use of `fmt.Printf` forbidden by pattern || use of `fmt.Prinln` forbidden by pattern
```

```suggestion
    text: use of `fmt.Printf` forbidden by pattern || use of `fmt.Println` forbidden by pattern
```
There is a typo here though, and it does not seem to matter, why is that? Replacing the text by garbage does matter, so it does something ...

---

I fixed typo.